test/integration/scheduler/framework_test.go

```diff
@@ -407,10 +407,17 @@ func (pp *PostFilterPlugin) PostFilter(ctx context.Context, state *framework.Cyc
 	if err != nil {
 		return nil, framework.NewStatus(framework.Error, err.Error())
 	}
+
 	ph := pp.fh.PreemptHandle()
 	for _, nodeInfo := range nodeInfos {
 		ph.RunFilterPlugins(ctx, state, pod, nodeInfo)
 	}
+	var nodes []*v1.Node
+	for _, nodeInfo := range nodeInfos {
+		nodes = append(nodes, nodeInfo.Node())
+	}
+	ph.RunScorePlugins(ctx, state, pod, nodes)
```

Comment:
Not sure if we need some caveat here or in `PluginsRunner` interface to make it clear that this is potentially for out-of-tree plugins only.

Reply:
Thanks for the comment.  I think I don't need comments here and `PluginsRunner` because the scheduler framework is essentially for out-of-tree plugins.

Reply:
okay, thanks!